### ISSUE TITLE
two spaces after 'rsyslogd:' in startup message

### ISSUE THREAD

**Issue by mbiebl** [rsyslog/rsyslog#2979]
Version: 8.37.0

When rsyslogd starts up or is restarted, it logs messages like this:
```
2018-08-26T00:00:42.313786+02:00 pluto rsyslogd:  [origin software="rsyslogd" swVersion="8.37.0" x-pid="638" x-info="http://www.rsyslog.com"] rsyslogd was HUPed
```
Note how there are two spaces after 'rsyslogd:'
This is inconsistent with how other messages are logged. Example
```
2018-09-01T00:01:35.624648+02:00 pluto anacron[18866]: Anacron 2.3 started on 2018-09-01
```
No two spaces after 'anacron[18866]:'

Apparently older versions of rsyslogd didn't have those two spaces, so this broke
https://salsa.debian.org/debian/rsyslog/blob/master/debian/rsyslog.logcheck.ignore.server

See https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=907755

**Comment by rgerhards** [comment]
thx for reporting. Looks like a pretty old regression...

**Comment by rgerhards** [comment]
also does not appear if I add

```
$ActionFileDefaultTemplate RSYSLOG_TraditionalFileFormat
```

To the config. @mbiebl can you provide some more info, best a debug log?

**Comment by mbiebl** [comment]
Tested with sysvinit and systemd, the behaviour is shown under both.
Config is attached 
[rsyslog.conf.txt](https://github.com/rsyslog/rsyslog/files/2353413/rsyslog.conf.txt)

**Comment by mbiebl** [comment]
debug log attached
[debug.log.txt](https://github.com/rsyslog/rsyslog/files/2353455/debug.log.txt)

```1308.623756773:main Q:Reg/w0  : omfile.c: omfile: write to stream, pData->pStrm 0x7fc6a40029c0, lenBuf 133, strt data Sep  5 17:28:28 debian rsyslogd:  [origin software="rsyslogd" swVersion="8.37.0" x-pid="1755" x-info="http://www.rsyslog.com"] s```

The trailing s is a bit weird, but this line shows the double spaces between rsyslogd: and [

**Comment by mbiebl** [comment]
@rgerhards Are you still unable to reproduce the issue? I can easily reproduce it with an up-to-date rsyslog and symply running `systemctl restart rsyslog.service`. I then get in /var/log/syslog:

```2019-06-21T13:45:26.752612+02:00 pluto systemd[1]: Stopped System Logging Service.
2019-06-21T13:45:26.756555+02:00 pluto systemd[1]: Starting System Logging Service...
2019-06-21T13:45:26.936465+02:00 pluto rsyslogd: imuxsock: Acquired UNIX socket '/run/systemd/journal/syslog' (fd 3) from systemd.  [v8.1905.0]
2019-06-21T13:45:26.936640+02:00 pluto rsyslogd:  [origin software="rsyslogd" swVersion="8.1905.0" x-pid="13027" x-info="https://www.rsyslog.com"] start
```

**Comment by rgerhards** [comment]
I thought I had fixed this. It is an incompatbility in imjournal IIRC.

**Comment by mbiebl** [comment]
Hm, odd. I'm not using imjournal here but imuxsock

**Comment by rgerhards** [comment]
I now remember the fix I did. That was a problem between our startup message format and journal. It's fixed. I will have a fresh look at the issue.

**Comment by rgerhards** [comment]
mhhh.. the sample is again the startup message; anyhow, will investigate.

**Comment by rgerhards** [comment]
Aha! I worked on this in September 2018 - have discovered the branch while preparing for updates. Now submitted as PR. That's probably why I thought it was fixed.

Anyhow, the old branch just fixes the double-space in startup messages. From the last postings, it looks like these are the prime (only?) issues. @mbiebl what do you think?

**Comment by mbiebl** [comment]
I think I've only ever seen those double spaces in the startup message.
The debian bug report is also only about the startup message.

**Comment by rgerhards** [comment]
thx - so I assume that this forgotten branch finally fixes it and will close after merge (the bug was quite obvious if you look at the PR). If new oddities show up, let's open a new issue for them. Sorry for the confusion...

**Comment by mbiebl** [comment]
To be specific: this are the three messages where I see the double spaces
```
Jul 02 17:04:01 pluto rsyslogd[637]:  [origin software="rsyslogd" swVersion="8.1905.0" x-pid="637" x-info="https://www.rsyslog.com"] rsyslogd was HUPed
Jul 02 17:05:11 pluto rsyslogd[637]:  [origin software="rsyslogd" swVersion="8.1905.0" x-pid="637" x-info="https://www.rsyslog.com"] exiting on signal 15.
Jul 02 17:05:11 pluto rsyslogd[12138]:  [origin software="rsyslogd" swVersion="8.1905.0" x-pid="12138" x-info="https://www.rsyslog.com"] start
```

**Comment by rgerhards** [comment]
..,. what matches the PR perfectly! Thx again.

**Comment by rgerhards** [comment]
fixed via #3736
